src/world_interface/sim_3d_interface.cpp

```diff
@@ -0,0 +1,306 @@
+#include "../Constants.h"
+#include "../Globals.h"
+#include "../Networking/websocket/WebSocketProtocol.h"
+#include "../Util.h"
+#include "../ar/read_landmarks.h"
+#include "../base64/base64_img.h"
+#include "../camera/Camera.h"
+#include "../camera/CameraConfig.h"
+#include "../kinematics/DiffDriveKinematics.h"
+#include "../log.h"
+#include "kinematic_common_interface.h"
+#include "world_interface.h"
+
+#include <atomic>
+#include <condition_variable>
+#include <mutex>
+#include <shared_mutex>
+#include <string>
+
+#include <nlohmann/json.hpp>
+#include <opencv2/core.hpp>
+
+using nlohmann::json;
+
+namespace {
+constexpr double MAX_WHEEL_VEL =
+	Constants::WHEEL_RADIUS * Constants::MAX_DRIVE_PWM / Constants::PWM_PER_RAD_PER_SEC;
```

Comment:
Oh, okay, I understand the maximum wheel velocity now - guess I just needed to see it.

Reply:
Now that I'm looking at it, I think this constant should probably be moved to `Constants.h`, even if it's calculated from other constants. Otherwise, the same calculation would be duplicated in multiple places. I'll add that right now.